tests/cases/trait/one-per-file

```diff
@@ -19,4 +19,5 @@ trait Bar
 
 ---MESSAGES---
 11:1 PSR1.Classes.ClassDeclaration.MultipleClasses
+11:1 Squiz.Classes.ClassFileName.NoMatch
```

Comment:
Regression? trait Foo does exist in the file

Reply:
See #37.

Reply:
Ah cool, makes sense 👍